core/src/main/java/com/novoda/noplayer/mediaplayer/CheckBufferHeartbeatCallback.java

```diff
@@ -4,9 +4,7 @@
 import com.novoda.noplayer.Player;
 import com.novoda.noplayer.VideoPosition;
 
-@SuppressWarnings("PMD.RedundantFieldInitializer")
-        // we're being very explicit with our default field values, not a bad thing!
-class CheckBufferHeartbeatCallback implements Heart.Heartbeat.Callback<Player> {
+public class CheckBufferHeartbeatCallback implements Heart.Heartbeat.Callback<Player> {
```

Comment:
Would make sense to move the `Buffer` interface defined here into `Heart.Heartbeat`? So instead of implementing this class, `BufferHeartbeatListener` can implement `Heart.Heartbeat.Buffer` and we can keep this one again as package protected?